hw/bsp/dialog_da1469x-dk-pro/da1469x_serial.py

```diff
@@ -35,6 +35,15 @@ def load(infile, uart, reset_script):
     except serial.SerialException:
         raise SystemExit("Failed to open serial port")
 
+    # drain serial port buffer
+    try:
+        while True:
+            data = ser.read(1)
+            if len(data) == 0:
+                break
+    except serial.SerialException:
+        raise SystemExit("Failed to open serial port")
+
```

Comment:
nit.  you're mixing two sets of changes here.  One to drain the port and the other to make the reset optional.

Reply:
done.